client/http/http.go

```diff
@@ -93,9 +97,16 @@ func (tc *TracedClient) Do(req *http.Request) (*http.Response, error) {
 	}
 
 	ext.HTTPStatusCode.Set(ht.Span(), uint16(rsp.StatusCode))
-	reqDurationMetrics.
-		WithLabelValues(req.Method, req.URL.Host, strconv.Itoa(rsp.StatusCode)).
-		Observe(time.Since(start).Seconds())
+	durationHistogram := reqDurationMetrics.WithLabelValues(req.Method, req.URL.Host, strconv.Itoa(rsp.StatusCode))
+
+	traceID, ok := wct.ExtractTraceID(req.Context())
```

Comment:
You can use [`opentracing-go.SpanFromContext()`](https://pkg.go.dev/github.com/opentracing/opentracing-go#SpanFromContext) instead to avoid adding another module.